app/services/witter.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const config = require('../../config');
+const logger = require('../logger');
+const errors = require('../errors');
+
+export async function weet() {
+  try {
+    const { data } = await axios.get(`${config.common.apiClient}/quotes/random`);
+    return data;
+  } catch (err) {
+    const error = errors.defaultError('There is an error in server');
+    return logger.error(error);
```

Comment:
Here I meant that you will use the logger to show the error, that you should not return, and that you will add a custom error, for example here you are using the `defaultError`, the idea is that you create a specific one for when integration with an API fails, for example, something like this:
```
logger.error (error.message);
return weetApiError(ErrorMessages.WEET_API_ERROR);
```